sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -64,6 +65,10 @@ def get(self, key, default=None):
             return self.__dict__[key]
         return default
 
+class PiiEntityDomainType(str, Enum):
+    """The different domains of PII entities that users can filter by"""
+    PHI = "PHI"
```

Comment:
can we include a comment for PHI about what it stands for?

Reply:
According to the guidance from KC:
> "Do not use abbreviations or acronyms in identifies. [...] It is more important for names to be readable than it is for them to be brief. I t is equally important not to use abbreviations and acronyms that are not generally understood"

Because of that, we should rename `PHI` to `ProtectedHealthInformation`

Reply:
I feel like `PHI` fits better in this case though, since we're abbreviating `PII`

Reply:
Even between us we've asked multiple times what `PHI` is. `Pii` is more common, so I strongly recommend changing it.
For .NET I will use the complete name

Reply:
ok, since .NET is using the full name, I'll make the change